NEWS.md

```diff
@@ -1,5 +1,11 @@
 # Packrat 0.9.0 (UNRELEASED)
 
+- Packrat now masks environment variables commonly used for Git service account
+  authentication from subprocesses used to install packages. This behavior can
+  be disabled by setting the option `packrat.unmask.git.service.vars` to `TRUE`.
+- Users can now mask arbitrary environment variables from the subprocess that
+  run package installation tasks, by setting the option `packrat.masked.envvars`
```

Comment:
The two settings use different terminology to talk about environment variables. Should we consistently use either `vars` or `envvars`?

Explain how this set of character vectors interacts with the `git.service.vars` option.

Reply:
Agreed, I should pick one. I think I'll use `envvars` because it's more precise, and there is [precedent](https://stat.ethz.ch/R-manual/R-devel/library/base/html/EnvVar.html).

Each option has an independent effect on the vector of variables to be is masked:
- The `git.service.envvars` governs whether that set of variables are added to the vector.
- And any user-specified variables are added to the vector.

I'll update the NEWS entry to make that clearer.